lib/src/gantt_default_week_header.dart

```diff
@@ -1,41 +1,56 @@
 import 'package:flutter/material.dart';
 
 class GanttChartDefaultWeekHeader extends StatelessWidget {
+  final Color? color;
+  final Color? backgroundColor;
+  final BoxBorder? border;
+  final WidgetBuilder? widgetBuilder;
+
   const GanttChartDefaultWeekHeader({
     Key? key,
     required this.weekDate,
+    this.color,
+    this.backgroundColor,
+    this.border,
+    this.widgetBuilder,
   }) : super(key: key);
   final DateTime weekDate;
 
   @override
   Widget build(BuildContext context) {
-    final bgColor = Colors.blue.shade900;
     return Container(
       padding: const EdgeInsetsDirectional.only(start: 8, top: 1, bottom: 1),
       decoration: BoxDecoration(
-        color: bgColor,
-        border: const BorderDirectional(
-          start: BorderSide(),
-          bottom: BorderSide(),
-        ),
-      ),
-      child: Center(
-        child: LayoutBuilder(builder: (context, constraints) {
-          String txt;
-          if (constraints.maxWidth < 50) {
-            txt = weekDate.month.toString();
-          } else if (constraints.maxWidth < 7 * 20) {
-            txt = '${weekDate.month}-${weekDate.year % 100}';
-          } else {
-            txt = '${weekDate.day}-${weekDate.month}-${weekDate.year}';
-          }
-
-          return Text(
-            txt,
-            style: const TextStyle(color: Colors.white),
-          );
-        }),
+        color: backgroundColor ?? defaultBackgroundColor,
+        border: border ?? defaultBorder,
       ),
+      child: widgetBuilder?.call(context) ?? defaultChild(context),
     );
   }
+
+  Color get defaultColor => Colors.white;
+
+  Color get defaultBackgroundColor => Colors.blue.shade900;
+
+  BoxBorder get defaultBorder => const BorderDirectional(
+        start: BorderSide(),
+        bottom: BorderSide(),
+      );
```

Comment:
can these be defined as static consts ?

Reply:
or they can be defined as `static Color defaultColor  = Colors.white;`
which will allow developers to do 
`GanttChartDefaultWeekHeader.defaultColor = SomeColor;`